swedbank-pay-design-guide-jekyll-theme.gemspec

```diff
@@ -2,21 +2,21 @@
 
 Gem::Specification.new do |spec|
   spec.name          = 'swedbank-pay-design-guide-jekyll-theme'
-  spec.version       = '1.5.1-article-no-main0001'
+  spec.version       = '1.5.2-conversion-to-gem-theme013'
```

Comment:
Can we have the version number sourced from `gitversion.json`, created by a new build step that gets its information from the `gitversion` build step and bundled up into the gem, somehow?

Reply:
I'm not sure how to accomplish this task.